package.json

```diff
@@ -86,7 +86,7 @@
     "babel-core": "^7.0.0-bridge.0",
     "babel-jest": "^27.5.1",
     "babel-loader": "^8.2.5",
-    "babel-plugin-styled-components": "^2.0.7",
+    "babel-plugin-styled-components": "2.0.6",
```

Comment:
Should this be changing?

Reply:
no I have it in another PR Ill  strip this out

Reply:
There was strange changes in snapshots made this PR
https://github.com/grommet/grommet/pull/6110

Reply:
taken out